auto-generated/cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/appstudio.redhat.com_v1alpha1_releaseplanadmission_rhtap-tools-rpa.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlanAdmission
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: "true"
+  name: rhtap-tools-rpa
+  namespace: rh-managed-rhtap-ser-tenant
+spec:
+  applications:
+  - tools
+  data:
+    images:
+      addGitShaTag: true
+      defaultTag: latest
+    mapping:
+      components:
+      - name: tools
+        repository: quay.io/redhat-appstudio/tools
+    pyxis:
+      secret: pyxis-production-secret
+      server: production
+  origin: rhtap-o11y-tenant
+  pipeline:
+    pipelineRef:
+      params:
+      - name: url
+        value: https://github.com/redhat-appstudio/release-service-catalog.git
+      - name: revision
+        value: production
+      - name: pathInRepo
+        value: pipelines/rhtap-service-push/rhtap-service-push.yaml
+      resolver: git
+    serviceAccountName: rhtap-servicerelease-sa
+  policy: rh-policy
```

Comment:
I think you are missing the `-o11y` part in the policy. For what I see in the integration test scenario below

Reply:
hmmm... I see `o11y` and `segment-bridge` using this policy for this part. Not sure which one is more appropriate.

Reply:
I see a bunch of recent PRs that moved back and forth with those configs for o11y:
https://github.com/redhat-appstudio/tenants-config/pull/223/files
https://github.com/redhat-appstudio/tenants-config/pull/225/files
https://github.com/redhat-appstudio/tenants-config/pull/227/files

Not sure what was the reasoning behind it, but I think we might be ok with starting this way and figure this out a little later.